Automattic/wp-calypso
# Pricing Grid: Implement storage options display

#### Proposed Changes

* This PR is one among several stacked PRs that implements the new pricing grid design. In https://github.com/Automattic/wp-calypso/pull/71372, we introduced a new signup flow /start/onboarding-2023-pricing-grid that is only available in dev and wpcalypso environments.
* The design for the pricing grid can be found in 2e991-pb. Please keep the Figma design open in a separate tab.
* The parent PR is https://github.com/Automattic/wp-calypso/pull/71511.
* This PR implements the Storage row in the features table.

<img width="1563" alt="Screenshot 2022-12-26 at 3 53 37 PM" src="https://user-images.githubusercontent.com/1269602/209537961-a09ab8fb-9b43-42d3-bac0-ab6025d46d66.png">

#### What Not To Test

* Responsive screens. It will be implemented in a separate PR. Right now, test in a full width desktop browser viewport.
* Non-en. The first launch will be `en` only.


#### Testing Instructions



*  Navigate to `/start/onboarding-2023-pricing-grid`.
* Check that the storage options are displayed as per the design in Figma. 
* Verify that the production `onboarding` signup flow at `/start` is not affected by these changes.

#### Pre-merge Checklist





Related to #
